customer-testing/src/main/java/no/unit/nva/customer/testing/CustomerDataGenerator.java

```diff
@@ -92,13 +92,20 @@ public static ChannelClaimDto randomChannelClaimDto() {
         return new ChannelClaimDto(randomChannel(), randomChannelConstraintDto());
     }
 
-    public static URI randomChannel() {
+    public static URI randomChannelOfType(String channelType) {
```

Comment:
I'd've preferred a fluent interface like `RandomChannel.new().withType(type)`…, but this is fine for now